File: exampleProjects/GoExample/src/de/sots/cellarsandcentaurs/domain/model/creature_type.go
package model

type CreatureType string

const (
	Beast     CreatureType = "BEAST"
	Humanoid  CreatureType = "HUMANOID"
	Dragon    CreatureType = "DRAGON"
	Undead    CreatureType = "UNDEAD"
	Fiend     CreatureType = "FIEND"
	Celestial CreatureType = "CELESTIAL"
)

func (ct CreatureType) String() string {
	return string(ct)
}

func (ct CreatureType) IsValid() bool {
	switch ct {
	case Beast, Humanoid, Dragon, Undead, Fiend, Celestial:
		return true
	default:
		return false
	}
}

File: exampleProjects/PythonExample/src/de/sots/cellarsandcentaurs/domain/models/creature_type.py
from enum import Enum

class CreatureType(Enum):
    HUMANOID = "humanoid"
    BEAST = "beast"
    UNDEAD = "undead"
    DRAGON = "dragon"
    GIANT = "giant"
    FIEND = "fiend"
    CELESTIAL = "celestial"
    ELEMENTAL = "elemental"
    ABERRATION = "aberration"
    CONSTRUCT = "construct"
    OOZE = "ooze"
    PLANT = "plant"
    MONSTROSITY = "monstrosity"
    FEY = "fey"

__all__ = ['CreatureType']